### PR TITLE
updated to build s390x image

### PR DESCRIPTION
**Pull request by ccorley**

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -20,8 +20,6 @@ RUN set -eux; \
 	esac; \
     if [ "$ASSET" = "ce" ] ; then \
         apk add --no-cache --virtual .build-deps curl wget tar ca-certificates && \
-        curl -fL "https://bintray.com/kong/kong-alpine-tar/download_file?file_path=kong-$KONG_VERSION.$arch.apk.tar.gz" -o /tmp/kong.tar.gz && \
-        echo "$KONG_SHA256  /tmp/kong.tar.gz" | sha256sum -c -; \
         apk del .build-deps; \
```
Comment (ccorley) [inline alpine/Dockerfile:23]: A final implementation would add the s390x arch as line 20 and not delete current lines 23 and 24. To get to that point, we would need to be able to push the s390x apk to bintray.com.

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 KONG_BUILD_TOOLS?=4.8.0
-BASE?=centos
+BASE?=alpine
```
Comment (ccorley) [inline Makefile:2]: Using alpine, as per the amd64 official docker image.

**Comment by ccorley** [comment]
The file kong.tar.gz is generated from kong-build-tools `make package-kong`, copied to the alpine directory and renamed.

File: alpine/Dockerfile (hunk 1 above)
Comment (dixonwhitmire) [reply alpine/Dockerfile:23]: Gotcha. 
So these PRs will let us build "locally" so to speak and we can then update to align with the standard process once we've got an artifact in bintray?

File: Makefile (hunk 2 above)
Comment (dixonwhitmire) [reply Makefile:2]: Can we pass this in as a "build argument" or do we need to override the default here. I wasn't sure if we would continue to "override" the default once the artifact is in bintray.

**Review by dixonwhitmire — APPROVED** [review]
If you can build it, it looks good to me :) I just had a few questions about what, if anything, in the PR would change once we have uploaded to bintray.

**Comment by ccorley** [comment]
Answering the right question - the apk specifically built for alpine would get pushed to bintray, but we would still need to override the BASE variable when building the kong image.

**Pull request merged**